xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/ClassEditSheet.xml

```diff
@@ -614,7 +614,9 @@ xcontext.put('propertyCustomDisplayer', new PropertyCustomDisplayer(xcontext))
       #set($queryString = 'wizard=true')
       #if(!$xwiki.exists($homePageRef))
         #set($homePageTitle = '$services.localization.render(''platform.appwithinminutes.appHomePageTitle'', [$doc.space])')
-        #set($queryString = "$queryString&amp;editor=inline&amp;template=AppWithinMinutes.LiveTableTemplate&amp;AppWithinMinutes.LiveTableClass_0_class=$escapetool.url($doc.fullName)&amp;title=$escapetool.url($homePageTitle)&amp;parent=AppWithinMinutes.WebHome")
+        #set($wikiHomePageRef = $services.model.resolveDocument('', 'default'))
```

Comment:
@Enygma2002 what @gabrielasmeria did was to follow the behaviour of the [logo link](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/src/main/resources/flamingo/companylogo.vm#L29) from the top menu bar. I'm against storing the configured home page in the parent field because the parent field won't be updated when the home page configuration changes. And saving (possibly) lots of pages when changing the home page configuration is not a good idea. Ideally we should be able to set the value of the parent field to a "token" that represents the configured home page whatever that may be. But since we don't have such a "token" the best option is IMO to store the default home page as parent. An administrator can redirect the default home page to the configured home page. He can also display the title of the configured home page in the title of the default home page using Velocity so that the breadcrumb looks nice.